modules/infra/api/gr_infra.h

```diff
@@ -18,6 +18,7 @@
 #define GR_IFACE_TYPE_UNDEF 0x0000
 #define GR_IFACE_TYPE_PORT 0x0001
 #define GR_IFACE_TYPE_VLAN 0x0002
+#define GR_IFACE_TYPE_LOOPBACK 0x0004
```

Comment:
Maybe we could use a value that does not risk being accidentally taken by another logical interface type? How about `0x1337` or something similar?

Reply:
Any arbitrary number may be used here.
GR_IFACE_TYPE_IPIP uses 0x003, that's why I took 0x004.